src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.card {
+  &__stars {
+    display: flex;
+    align-items: end;
+    justify-content: space-between;
+    line-height: 14px;
+
+    &-reviews {
+      margin: 0;
+      font-size: 10px;
+      color: $title-color;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 4px;
+  width: 96px;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--4 .stars__star:nth-last-child(1n + 2) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
Code Style issue: According to BEM methodology, it's not recommended to use nested selectors like `.stars--4 .stars__star:nth-last-child(1n + 2)`. Instead, you can create a modifier for the `.stars__star` element, like `.stars__star--active`, and apply it directly to the active stars.